src/librbd/internal.cc

```diff
@@ -2217,6 +2218,15 @@ int mirror_image_disable_internal(ImageCtx *ictx, bool force,
         return -EBUSY;
       }
 
+      cls::rbd::GroupSpec s;
+      r = cls_client::image_get_group(&io_ctx, header_oid, s);
+      if ((!s.group_id.empty()) && (s.pool_id != -1)) {
+	lderr(cct) << "image is in a consistency group - not removing" << dendl;
+	ictx->owner_lock.put_read();
+	ictx->state->close();
+	return -EMLINK;
```

Comment:
There is a discussion earlier in this pull request. It's used for nicer output. If it's busy then we report that somebody is using it, if it's EMLINK we report that it belongs to a consistency group and specify what consistency group it belongs to.